compose/service/values/shared.go

```diff
@@ -0,0 +1,37 @@
+package values
+
+import (
+	"regexp"
+	"time"
+)
+
+const (
+	strBoolTrue  = "1"
+	strBoolFalse = "0"
+
+	datetimeInputFormatDate = "2006-01-02"
+	datetimeInputFormatTime = "15:04:05"
+	datetimeInputFormatFull = time.RFC3339
+
+	fieldOpt_Datetime_onlyDate         = "onlyDate"
+	fieldOpt_Datetime_onlyTime         = "onlyTime"
+	fieldOpt_Datetime_onlyFutureValues = "onlyFutureValues"
+	fieldOpt_Datetime_onlyPastValues   = "onlyPastValues"
+
+	fieldOpt_Number_precision = "precision"
+
+	fieldOpt_Url_onlySecure = "onlySecure"
+)
+
+var (
+	// value resembles something that can be true
+	truthy = regexp.MustCompile(`^(t(rue)?|y(es)?|1)$`)
```

Comment:
add `i` (case insensitive) flag for good measure

Reply:
we do  it on the value.